core/util/eventtarget.js

```diff
@@ -0,0 +1,45 @@
+/*
+ * noVNC: HTML5 VNC client
+ * Copyright 2017 Pierre Ossman for Cendio AB
+ * Licensed under MPL 2.0 (see LICENSE.txt)
+ *
+ * See README.md for usage and integration instructions.
+ */
+
+export var EventTargetMixin = {
```

Comment:
just `export default` this

Reply:
Not sure why I didn't in the first place...